transport-native-epoll/src/main/java/io/netty/channel/epoll/EpollEventLoop.java

```diff
@@ -391,7 +391,11 @@ protected void run() {
                     events.increase();
                 }
             } catch (Throwable t) {
-                handleLoopException(t);
+                if (t instanceof ThreadDeath) {
```

Comment:
Please refactor to:

```java
try {
    ...
} catch (ThreadDeath td) {
    throw (ThreadDeath) td;
} catch (Throwable t) {
    handleLoopException(t);
}
```

Some for all other places.

Reply:
Sure, refactored.